rskj-core/src/main/java/co/rsk/peg/utils/BrigeEventLoggerLegacyImpl.java

```diff
@@ -128,14 +127,21 @@ public void logCommitFederation(Block executionBlock, Federation oldFederation,
         this.logs.add(new LogInfo(BRIDGE_CONTRACT_ADDRESS, topics, data));
     }
 
+    @Override
+    public void logCommitFederationFailure(Block executionBlock, Federation proposedFederation) {
```

Comment:
Add tests for this please.

Reply:
https://github.com/rsksmart/rskj/pull/2776/files#diff-e8b26925746c6e67511b4dd8dfe7b1d8787e41d764cf8f0fb2a2c1c2544a00f0R328 ?

Reply:
updated!